test/e2e/suite_test.go

```diff
@@ -41,7 +41,7 @@ func TestAPIs(t *testing.T) {
 }
 
 var _ = ginkgo.BeforeSuite(func() {
-	k8sClient = framework.CreateClientUsingCluster()
+	k8sClient = util.CreateClientUsingCluster()
 	ctx = context.Background()
-	framework.KueueReadyForTesting(k8sClient)
+	util.KueueReadyForTesting(k8sClient)
```

Comment:
these two functions are not used in integration tests, so maybe you can move them here into the suite directly.

If we later see that we have to create more E2E suites, we can move them to a new `test/e2e/framework`